Examples/generics.ol

```diff
@@ -0,0 +1,28 @@
+class Main is
+    this is
+        var list: List<Dictionary<String, Pair<Integer, String>>> = []
+        list.Append({})
+        var pair = Pair(42, "Amogus")
+
+        list.Get(0).Set("sus", pair)
+        IO().WriteLine(list.Get(0).Search(pair))
+    end
+end
+
+class Pair<K, V> is
+    var key: K = null
+    var value: V = null
```

Comment:
I guess we need to think of `null` values.
There are none yet, I just added here to make the syntax analyzer consume it